app/src/main/java/com/github/se/icebreakrr/MainActivity.kt

```diff
@@ -372,9 +372,7 @@ fun IcebreakrrApp(
           tagsViewModel = tagsViewModel,
           permissionManager = permissionManager)
 
-  val startDestination =
-      if (isTesting) Route.AROUND_YOU
-      else (if (currentUser != null) Route.AROUND_YOU else Route.AUTH)
+  val startDestination = if (isTesting) Route.AROUND_YOU else Route.AUTH
```

Comment:
Here it would be better to keep the previous code. It is a cool feature to skip the login screen when we have already signed in and that we are re-entering the app

Reply:
We can discuss